draft-mouris-cfrg-mastic.md

```diff
@@ -237,9 +285,98 @@ This document uses the following terms as defined in {{!VDAF}}:
 "prep share", and
 "report".
 
-Mastic uses finite fields as specified in {{Section 6.1 of !VDAF}}. We usually
-denote a finite field by `F` and its Python class object, a subclass of
-`Field`, as `field: type[F]`.
+Mastic uses finite fields of prime order as specified in {{Section 6.1 of
+  !VDAF}}. We usually denote a finite field by `F` and its Python class object,
+  a subclass of `Field`, as `field: type[F]`. The following basic API functions
+  referenced in this document are as defined therein:
+
+* `Field(integer: int) -> Field` returns an integer represented as a field
+  element. The value of integer MUST be non-negative and less than
+  Field.MODULUS.
+* `Field.as_unsigned() -> int` returns the integer representation of a field
+  element.
+* `Field.encode_vec(vec: list[Field]) -> bytes` encodes a vector of field
+  elements `vec` as a byte string.
+* `Field.zeros(length: int) -> list[Self]` returns a vector of zeros of size
+  `length`.
+
+The following parameters are also referenced:
+
+  | Parameter           | Description                                           | Value                   |
+  |:--------------------|:------------------------------------------------------|:------------------------|
+  | `field: type[F]`    | class object for the field ({{Section 6.1 of !VDAF}}) | set by constructor      |
+  | `Field.MODULUS: int`| prime modulus defining the field arithmetic           | to be defined           |
+  {: #field-params title="Basic Field parameters."}
+
+
+Mastic uses Extendable Output Functions (XOFs) as specified in {{Section 6.2 of
+  !VDAF}}. The following XOF API functions referenced in this document are as
+  defined therein:
+
+  * `XofFixedKeyAes128(seed: bytes, dst: bytes, binder: bytes) -> Xof` is a XOF
+    based on a circular collision-resistant hash function from fixed-key AES that
+    is initialized with a given seed, a domain separation tag, and binder string,
+    and returns a `Xof` instance. The length of `seed` will typically be
+    SEED_SIZE, but some XOFs may support multiple sizes. The `seed` MUST be
+    generated securely (i.e., it is either the output of a CSPRNG or a previous
+    invocation of the XOF). The length of the domain separation string `dst`
+    passed to XofTurboShake128 MUST NOT exceed 65535 bytes.
+  * `XofTurboShake128(seed: bytes, dst: bytes, binder: bytes) -> Xof` is a XOF
+    wrapper for TurboSHAKE128 that is initialized with a given seed, a domain
+    separation tag, and binder string, and returns a `Xof` instance. The length of
+    `seed` will typically be SEED_SIZE, but some XOFs may support multiple sizes.
+    The `seed` MUST be generated securely (i.e., it is either the output of a
+    CSPRNG or a previous invocation of the XOF). The length of the domain
+    separation string `dst` passed to XofTurboShake128 MUST NOT exceed 65535
+    bytes.
+  * `Xof.next(length: int) -> bytes` returns the next `length` bytes of output of
+    a `Xof` instance.
```

Comment:
I'm also in favor of not listing these as they're subject to change in VDAF so we'd also need to update accordingly. I like what you have above:
```
Mastic uses Extendable Output Functions (XOFs) as specified in {{Section 6.2 of !VDAF}}. 
```

Reply:
If the API changes, so will our code that references them in this .md. With >12 iterations already, are you expecting major changes in the !VDAF API soon?